lib/utils/config/definitions.js

```diff
@@ -1495,8 +1495,6 @@ define('package-lock', {
     If set to false, then ignore \`package-lock.json\` files when installing.
     This will also prevent _writing_ \`package-lock.json\` if \`save\` is
     true.
-
-    This configuration does not affect \`npm ci\`.
```

Comment:
The thinking here is that now that this isn't listed as a flag on `npm ci` the caveat isn't needed.